ngraph/core/include/ngraph/op/multiply.hpp

```diff
@@ -89,9 +54,7 @@ namespace ngraph
             };
         } // namespace v1
 
-        NGRAPH_SUPPRESS_DEPRECATED_START
-        using v0::Multiply;
-        NGRAPH_SUPPRESS_DEPRECATED_END
+        using v1::Multiply;
```

Comment:
We can set BinaryElementwiseArithmetic auto-broadcast to default NUMPY and get rid of all the default ctors in the child classes.
```
 Multiply()
                    : util::BinaryElementwiseArithmetic(AutoBroadcastSpec::NUMPY)
                {
                }
```

Reply:
Default constructors are required from `ngraph/core/include/ngraph/factory.hpp`
`cpp
 // \brief Get the default factory for DERIVED_TYPE. Specialize as needed.
        template <typename DERIVED_TYPE>
        static Factory get_default_factory()
        {
            return []() { return new DERIVED_TYPE(); };
        }
`